e2e/cypress/integration/system_console/lock_teammate_name_display_spec.js

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+describe('System Console', () => {
+    it('MM-19309 Allow System Admins to control Teammate Name Display at the system level', () => {
+        cy.apiLogin('sysadmin');
+        cy.visit('/admin_console/site_config/users_and_teams');
+        cy.findByTestId('TeamSettings.TeammateNameDisplaydropdown').select('nickname_full_name');
+        cy.findByTestId('TeamSettings.LockTeammateNameDisplayfalse').click();
+        cy.get('#saveSetting').click();
+
```

Comment:
Can you also add a step to verify the helper text displayed under the field Lock Teammate Name Display for all users. It should contain the following text:
`When true, disables users' ability to change settings under Main Menu > Account Settings > Display > Teammate Name Display.`

Reply:
Done